crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -513,6 +580,91 @@ fn legend_ui(ctx: &ViewerContext<'_>, space_view_id: SpaceViewId, ui: &mut egui:
         });
 }
 
+fn axis_ui(
+    ctx: &ViewerContext<'_>,
+    space_view_id: SpaceViewId,
+    ui: &mut egui::Ui,
+    state: &TimeSeriesSpaceViewState,
+) {
+    // TODO(jleibs): use editors
+
+    let (
+        re_types::blueprint::archetypes::AxisY {
+            range,
+            zoom_behavior,
+        },
+        blueprint_path,
+    ) = query_space_view_sub_archetype(ctx, space_view_id);
+
+    ctx.re_ui
+        .selection_grid(ui, "time_series_selection_ui_axis_y")
+        .show(ui, |ui| {
+            ctx.re_ui.grid_left_hand_label(ui, "Axis Y");
+
+            ui.vertical(|ui| {
+                let mut auto_range = range.is_none();
+
+                ui.horizontal(|ui| {
+                    ctx.re_ui
+                        .radio_value(ui, &mut auto_range, true, "Auto")
+                        .on_hover_text("Automatically adjust the Y axis to fit the data.");
+                    ctx.re_ui
+                        .radio_value(ui, &mut auto_range, false, "Override")
+                        .on_hover_text("Manually specify a min and max Y value.");
+                });
+
+                if !auto_range {
+                    // TODO(jleibs): Initial range needs to come from the real data.
+                    let mut range_edit =
+                        range.unwrap_or_else(|| range.unwrap_or(Range1D(state.saved_y_axis_range)));
+
+                    ui.horizontal(|ui| {
+                        let speed_min = (range_edit.0[0].abs() * 0.01).at_least(0.001);
+                        let speed_max = (range_edit.0[1].abs() * 0.01).at_least(0.001);
+                        ui.label("Min");
+                        ui.add(egui::DragValue::new(&mut range_edit.0[0]).speed(speed_min));
+                        ui.label("Max");
+                        ui.add(egui::DragValue::new(&mut range_edit.0[1]).speed(speed_max));
```

Comment:
inverted ranges don't seem to work, we should block them

Reply:
Looks like you fixed this.